exposed-core/src/main/kotlin/org/jetbrains/exposed/sql/transactions/TransactionScope.kt

```diff
@@ -5,31 +5,18 @@ import org.jetbrains.exposed.sql.Transaction
 import kotlin.properties.ReadWriteProperty
 import kotlin.reflect.KProperty
 
-@Suppress("UNCHECKED_CAST")
-fun <T:Any> transactionScope(init: Transaction.() -> T) = TransactionStore(init) as ReadWriteProperty<Any?, T>
-fun <T:Any> nullableTransactionScope() = TransactionStore<T>()
```

Comment:
`nullableTransactionScope ` was not used.  Ironically this is the only function that would leak `TransactionStore`, since `transactionScope` is manually cast to a `ReadWriteProperty`.  I am fine leaving a `@Deprecated` `nullableTransactionScope` call that has type `ReadWriteProperty` and returns a delegate that always returns `null`.  @Tapac do you think this is necessary?